src/lwc/rd2EditPaymentInformationModal/rd2EditPaymentInformationModal.js

```diff
@@ -190,7 +190,7 @@ export default class Rd2EditCreditCardModal extends LightningElement {
         if (!isNull(response) && !isUndefined(response)
             && !isNull(response.body) && !isUndefined(response.body)
         ) {
-            const responseBody = JSON.parse(response.body);
+            const responseBody = (typeof response.body === "object") ? response.body : JSON.parse(response.body);
```

Comment:
Is this change only to support the Jest test? That's fine, I'm just curious why it wasn't needed before now!

Reply:
Nice catch, The type check is now reverted back to original. I added the type check because I thought jest only support mock response in JSON and not String.